rules/imagemagick.json

```
@@ -56,7 +56,7 @@
         ]
       },
       {
-        "packages": ["ImageMagick", "ImageMagick-c++"],
+        "packages": ["ImageMagick", "ImageMagick-c++", "ImageMagick-c++-devel", "ImageMagick-devel"],
```

Just for simplification, can this be reduced to the devel packages only?
```suggestion
        "packages": ["ImageMagick-c++-devel", "ImageMagick-devel"],
```